### ISSUE TITLE
Background expansion icon is not visible when it is disabled by default

### ISSUE THREAD

**Issue by jsmaniac** [racket/drracket#116]
When I install the latest snapshot (20170525-cc25a27f87) on a fresh Ubuntu 14.04.4 VirtualBox VM, upon the first start, the background expansion icon (the little red circle in the lower right corner) is not visible.

![no-icon](https://cloud.githubusercontent.com/assets/299872/26454669/0c716b3e-4168-11e7-9166-abcdec2ed271.png)

Even after selecting a language with <kbd>ctrl</kbd>+<kbd>L</kbd>, and pressing <kbd>F5</kbd> to run the program, the icon still does not show up. Same thing after typing an expression in the definitions window. DrRacket has to be restarted for the icon to show up.

**Comment by rfindler** [comment]
does it appear if you edit the definitions window (say, add a space and then delete it)?

**Comment by jsmaniac** [comment]
Nope, it does not appear after adding and removing a space (if background expansion is left in the default "disabled" state).

If I simply choose the language and restart DrRacket, then the red circle shows up after the restart.
